.gitignore

```diff
@@ -166,3 +166,5 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.\*
+
+.env.test
```

Comment:
It would be a good idea to check how many of the files listed in here are actually present in your project. If they are not there (such as the coverage folder, `logs`, `*.logs` etc), they can be removed from this file. If youre not sure, look in your IDE for the file/folder in question and if it isnt there, it can be removed from here